### PR TITLE
adding cluster-reader aggregation

### PR DESCRIPTION
**Pull request by kalexand-rh**
@mrogers950, will you PTAL?

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@mrogers950, will you PTAL?

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

File: modules/proc_creating-aggregated-cluster-role.adoc [hunk 1]
```diff
@@ -78,6 +79,7 @@ that represent the permissions that are granted to the role. For example, apply
 read and write permissions to the admin and edit roles and only read permission
 to the view role. 
 <8> Specify this label to grant permissions to the view default role. 
+<9> Specify this label to grant permissions to the cluster-reader default role.
```
Comment (mburke5678) [inline modules/proc_creating-aggregated-cluster-role.adoc]: should cluster-reader have mark-up, such as *cluster-reader*?

**Comment by mburke5678** [comment]
@kalexand-rh A nit squared. Otherwise LGTM

File: modules/proc_creating-aggregated-cluster-role.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/proc_creating-aggregated-cluster-role.adoc]: For the 3.x stream, yes.

**Comment by kalexand-rh** [comment]
Thank you @mburke5678!

**Pull request merged**